### ISSUE TITLE
.use() doesn't exist

### ISSUE THREAD

**Issue by Michaos** [AppsFlyerSDK/AppsFlyer-Segment-Integration#7]
Everything is in the title.

2. Next, declare AppsFlyer’s integration in your Analytics instance:
```
Analytics analytics = new Analytics.Builder(this, "Your-Segment-Write-Key")
        .use(AppsflyerIntegration.FACTORY)
        ...
        .build();

```

**Comment by shachar-af** [comment]
@Michaos can you please elaborate on what is the issue you're experiencing?
Are you getting some build/compilation/runtime error?

**Comment by shachar-af** [comment]
What version of `com.segment.analytics.Analytics` are you using? 
We are using `compile 'com.segment.analytics.android:analytics:4.+'` in our sample app's build.gradle file, which pulls out `analytics-4.2.6.jar`. This library does include this API:
`public Builder use(Integration.Factory factory);`

(refer to [the source code](https://github.com/segmentio/analytics-android/blob/bedd2a19cfd105728c5f1011fce2d6b29c9ac1f0/analytics/src/main/java/com/segment/analytics/Analytics.java) - at line 1137)

**Comment by Michaos** [comment]
Oh ok, didn't added compile 'com.segment.analytics.android:analytics:4.+'
I thought compile 'com.segment.analytics.android:analytics-core:3.1.7 was enough
thanks

**Comment by Michaos** [comment]
I still got issue with 
But I think it is mostly linked with other segment integration

```
compile 'com.segment.analytics.android:analytics:4.2.4'
compile 'com.appsflyer:segment-android-integration:1.5'
/*compile 'com.segment.analytics.android:analytics-core:3.1.7'
 *compile 'com.segment.analytics.android:analytics-integration-amplitude:3.4.0'
 *compile('com.segment.analytics.android:analytics-integration-tapstream:3.4.0') {
 *    exclude module: 'play-services-ads'
 *    exclude group: 'org.json', module: 'json'
 *}*/
```

Does not provoc the error, but if uncommented, it doesn't compile and tells me
error: cannot find symbol method use(Factory) when I try to compile

**Comment by shachar-af** [comment]
well, the same method also exists on the 4.2.4 version which you are declaring on the dependencies block.
So I would guess you should try running these commands on your project directory, which should clear you gradle dependencies cache:
```
rm -rf ~/.gradle/caches ; ./gradlew --refresh-dependencies ; ./gradlew clean build
```
Also, I would advise you to use our latest version available (for this repo - 1.6), and Segment's latest analytics library version (4.2.6).

Finally, you're more than welcome to use the sample app (in this repo), which demonstrates the integration.
